src/riak_kv_ts_error_msgs.hrl

```diff
@@ -108,3 +108,9 @@
      "Where clause when a range is not specified, but more than one was ",
      "specified.">>
 ).
+
+-define(
+    E_MISSING_COL_IN_GROUP_BY(FieldName, TableName),
+    iolist_to_binary(io_lib:format(
+        "Error in group by clause, column '~ts' does not exist in table table ~ts", [FieldName, TableName]))
```

Comment:
I can shorten this to "Column '~s' in GROUP BY clause does not exist in table '~ts'". (Or at least, delete one word "table" that appears in duplicate there.)